README.md

```diff
@@ -1,68 +1,102 @@
 
-Movie object format:
+* [ ] Now that we have access to setMovies, made sure the updated list of movies is saved to our global state.
 
-```
-{
-  id: 5,
-  title: 'Tombstone',
-  director: 'George P. Cosmatos',
-  metascore: 89,
-  stars: ['Kurt Russell', 'Bill Paxton', 'Sam Elliot'],
-}
-```
+* [ ] Redirect the user to the currently edited movie's indivisual info page.
 
-#### Deleting A Movie:
+### Deleting a Movie
+> *You added in a CRUD feature! Good job! Now let's get deleted squared away...*
 
-- Add a delete button in the movie component that makes a DELETE request
-- When the call comes back successfully, route the user to `/` where they will see the updated movie list without the deleted movie
+* [ ] Identify the component that holds the "button" needed for deletion. Add an event hanlder to that button.
 
-#### Adding A Movie:
+* [ ] Build your new event handler to make the request needed to delete the currentlt viewed movie. Observe what is returned from the request.
 
-- Add a route at the path `/add-movie`
-- Create a component with a form to add a new movie
-- Each created movie should have the following format (notice the array of strings - this will test your JS skills, so work through it methodically)
-- The form should make a POST request to the server when submitted
-- When the call comes back successfully, reset your form state and route the user to `/`
+* [ ] You will once again need to keep the server and state data in sync. In the `App.js` class, complete complete the deleteMovie function so that it recieves a movie id and sets movies to be equal to all movies BUT the movie with the indicated id.
 
-Movie object format:
+* [ ] Make the changes needed to give the correct component addes to your deleteMovie function.
 
+* [ ] Call deleteMovie at the approprate time and redirect the user to the `/movies` route.
+
+### Adding a Movie
+> *Alright! You ready! Let's see you use the skills of the previous steps to build a crud function from start to finish.*
+
+* [ ] Use `EditMovieForm.js` as a model to build an `AddMovieForm` component. The component should hold all the attributes of a new movie in local state as defined in the rest of application.
+
+* [ ] Add in a route that allows access to the add Movie button.
+
+* [ ] Locate the part of the ui that should navigate to your new AddMovieForm. Make that button redirect to your new component.
+
+* [ ] On submission, run the approprate request for adding a movie with the component's state values.
+
+* [ ] Make sure your component has access to and runs and modifications needed to global state and redirects to `/movies` after creation.
+
+### Stretch goals
+- Make the added DeleteMovieModal appear and be reacted to before deletion occurs.
+- Add in add to favorites functionality. When the favorite button is pushed in the `Movie` component, make sure that when the favorite button is pushed, the id and name of the currently viewed into the favorite state slice in `App.js.`
```

Comment:
"favorite button is pushed" is written twice 😂

Also, the last part of that sentence needs to be reworded:

"add the id and name of the currently viewed movie into the favorite state slice in App.js."